.circleci/config.yml

```diff
@@ -278,23 +246,17 @@ jobs:
       - checkout
       - *git-submodules
       - *attach-to-workspace
-      - run:
-          name: Prepare database & config files
-          command: |
-            cp config/examples/*.yml config/
-            bundle exec rake db:create db:test:prepare
       - run:
           name: Rspec tests
-          command: |
-            bundle exec rspec --format progress $(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)
+          command: make test-rake JOB="integrate:rspec"
       - *store-junit-test-results
       - *store-test-artifacts
       - *store-log-artifacts
       - *upload-coverage
 
   cucumber:
-    parallelism: 20
-    resource_class: small
+    parallelism: 40
+    resource_class: medium
```

Comment:
How much does this improve the build time?

Reply:
hmmm - i think the `medium` might have been left over from something else I was trying to test.. verifying now